src/SimpleGraphs/overrides/operators.jl

```diff
@@ -0,0 +1,192 @@
+# override for performance
+function reverse(g::SimpleDiGraph)
```

Comment:
This is where we override the general `src/operatots.jl` functions for SimpleGraph-specific optimizations. We also add (and export from SimpleGraphs) an in-place `reverse!` for SimpleDiGraphs.